### PR TITLE
added a requirement to mantain the manual

### LINKED ISSUE #46: Mantain the Game Manual
We need to have a clear Game Manual.
This must be a document explaining the rules in a clear and easy to understand way, that encompasses any possible game situation.

This manual is on the Wiki (https://github.com/kenan-rhoton/card-game/wiki/0.-Troll:-Card-Game-Rules). 
It is **very important** that **everybody** reads it (otherwise I can't see how any rule can be correctly implemented). Also, any opinion regarding how to organize it or its clarity will be very well recieved.

### PR DESCRIPTION
**Pull request by Masclins**
**Description**

Since now the manual has a better format, it can be easily mantainable by including any new change.
I included a requirement for PR on CONTRIBUTING.

**Issues Resolved**

This would resolve #46 since it would be mantained as we keep developing.

**Details**

Maybe it would be interesting to add a reminder of this unto the PullRequest Template (and maybe even the rest of aspects. May we include a checklist on the PR template?)

### CODE REVIEW

**Comment by kenan-rhoton** [comment]
Please do add this to the PR template 🙃

**Comment by Masclins** [comment]
This = Checklist?

**Comment by kenan-rhoton** [comment]
Yes, the Checklist

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -13,3 +13,12 @@ More in-depth explanation about the code submitted and its purpose.
 **Hotspots**
 
 Special areas of code the reviewers should pay attention to.
+
+**Checklist for contribution requirements**
+
+- [ ] The tests pass
+- [ ] The code has tests
+- [ ] The tests are well-designed
+- [ ] The code is redeable
```
Comment (kenan-rhoton) [inline PULL_REQUEST_TEMPLATE.md]: `readable`is misspelled 🙃

**Review by kenan-rhoton — APPROVED** [review]

**Pull request merged**